example/android/app/build.gradle

```diff
@@ -100,6 +100,6 @@ android {
 
 dependencies {
     compile "com.android.support:appcompat-v7:23.4.0"
-    compile "com.facebook.react:react-native:0.29.0"  // From node_modules
+    compile "com.facebook.react:react-native:+"  // From node_modules
```

Comment:
hmm it's what react-native itself does.

This is just for the example app, but I'd love to see if there's a better way to define this dependency for the package itself as react-native moves so fast.